src/main/java/org/takes/facets/fork/FkHitRefresh.java

```diff
@@ -180,4 +173,15 @@ private void touch() throws IOException {
         }
     }
 
+    /**
+     * Create the file to touch.
+     * @return The file to touch
+     * @throws IOException If fails
+     */
+    private static File createFileToTouch() throws IOException {
```

Comment:
@essobedo check this out: http://www.yegor256.com/images/books/elegant-objects/seven-pages.pdf